src/legacy/ui/public/flyout/flyout_session.tsx

```diff
@@ -104,9 +105,11 @@ export function openFlyout(
   };
 
   ReactDOM.render(
-    <EuiFlyout {...flyoutProps} onClose={onClose}>
-      {flyoutChildren}
-    </EuiFlyout>,
+    <I18nProvider>
```

Comment:
Hey @flash1293, can you please replace `I18nProvider` with `EuiContext`? 

We shouldn't use `I18nProvider` directly in Kibana anymore, see `[ANNOUNCE] New I18nContext and EUI i18n` email thread in Kibana dev mailing list.

Reply:
@azasypkin ah, my bad. I will fix it in a new PR :+1: It seems like the packages/i18n readme isn't updated to reflect this yet (this is where I looked), is there a specific reason for this? If not I could also add this.

Reply:
> It seems like the packages/i18n readme isn't updated to reflect this yet (this is where I looked), is there a specific reason for this? If not I could also add this.  If not I could also add this.

That would be great! We should probably add that somewhere to main contribution docs instead of `packages/i18n` since `I18nContext` is exposed by the Kibana itself (`packages/i18n` + `@elastic/eui`).

P.S. change typo in my first post `EuiContext` ---> `I18nContext`.